pkg/operator/encryption/crypto/rsa.go

```diff
@@ -0,0 +1,76 @@
+package crypto
+
+import (
+	"crypto/rand"
+	"crypto/rsa"
+	"crypto/x509"
+	"encoding/pem"
+	"fmt"
+	"reflect"
+
+	"k8s.io/client-go/util/keyutil"
+)
+
+// Size for generated RSA key pairs
+const keySize = 4096
```

Comment:
Any particular reason we're bumping this to 4096 vs 2048 which is currently used in both operators? I'm mainly asking about backwards compatibility?